### DISCUSSION TITLE
Feed update

### DISCUSSION THREAD

**Discussion by mohaidoss** [community/community#19256] [Other Feature Feedback, Questions, & Ideas] [answered]
The "For You" feed has the same posts for 4 days now, maybe try to make it recommand different things every day ?

**Comment by dendcode** [comment]
it should be updated

**Comment by khzg** [comment]


**Reply by Svvaladez** [reply]
I’m having trouble posting, they really in me they locked me out of all my accounts, change my passwords, they’re from India using Tar, right now they don’t let me summit these answers, I have to be quick, I don’t know what to do, I called authorities but these ppl are quick, I’m about to lose it, they’re also trying to steal my sisters results from a genetic study cause she has cancer, they have no heart

**Comment by melvincwng** [comment]
@mohaidoss, try following new developers and starring new repositories to "spice up" the recommendations in the 'For You' page.

Hope this clarifies.

**Reply by mohaidoss** [reply]
Has almost no effect, it showed 3 new posts, that are still on top of the feed for now 5 days... I have at least followed 6 new accounts and starred 7 new projects. 

If the feed is called "For you", I expect it to be like tiktok or instagram feed, with less volatility of course..

**Reply by CryptonightCanada** [reply]
> @mohaidoss, try following new developers and starring new repositories to  "Spice Up" the _Recommendations_ in the 'For You' page.

> 
> Hope this clarifies.

> 

> 

> 💡

**Comment by Svvaladez** [comment]
https://github.com/orgs/github-community/discussions/new

**Comment by Svvaladez** [comment]
I’m sorry guys if I don’t answer quick, it’s hard for me to even read the messages, I’m not ignoring anyone, but I’m trying, to tell y’all I haven’t seen one cent of the profits, they’re getting it, they even changed my default pic from twitter, it’s a lot, I woke up one morning and my vehicle is gone from my driveway, my kids are staying with their dad due to this, then I try to get new numbers and they internally locked my devices when I paid in full for them, they own T mobile and metro stores, so I go to the apple store and no luck, I didn’t have password to my Apple ID cause they change it, so they couldn’t verify me but I had my I’d, when I call the bank they don’t give me information for security reasons but they allow them, I don’t understand, they had my fiancée arrested for a minor misdemeanor, sentenced to 6 months county time, some how his release date isn’t until 04/2028, they’re developers for jpay and securus so when I try to send him money they block me, I haven’t talked to him, and I don’t have a way to visit because I don’t have transportation and when I try to report they log me off, when I do fix my emails they right away change password and link my emails to them, this has been very hard, I don’t know how I’m still hanging on

**Comment by surajsahani** [comment]
"For You" suggestion feed show result based on your activity just like ads shown on a website with a google search.
 Hmmm... so how should I make my feed show me the best result?
Simple answer
1. Start your like repositories, fork repository push code whom you are working, your tech stack, explore tech which you like
2. From the next day ai will automatically show the optimized results.

**Reply by mohaidoss** [reply]
You got it wrong. I'm not talking here about if the recommendations are correct or not. The "For You" feed has no new content (either related to my activity or not) on it.
Regarding your example, that's like getting one exact same ad over and over.

**Reply by surajsahani** [reply]
I understand, that there is no new content because of no new activity. it will grow as you increase your activity just exploring more repositories will also work. try that out if it won't work you can create a bug issue on GitHub

Here are a few topics I'll suggest please  give it a try

1. Please go to your start topics and suggest add suggestions on a few topics. based on that suggestion GitHub feed will show you result

Attaching an image of how it should look.
you can easily find the stars repository tab by going to profile, than navigating to stars. 
![Screenshot from 2022-06-30 20-01-31](https://user-images.githubusercontent.com/22853459/176704356-9286d862-2e90-499b-9f67-0995f992d312.png)

**Reply by mohaidoss** [reply]
That's not how a feed is supposed to work... And still increasing my activity has no effect on how the algorithm works, just look up my comment an hour ago

**Reply by surajsahani** [reply]
Also, you can set email newsletter to daily from where it will show you most optimized result
https://github.com/explore/email

**Reply by mohaidoss** [reply]
> I understand, that there is no new content because of no new activity. it will grow as you increase your activity just exploring more repositories will also work. try that out if it won't work you can create a bug issue on GitHub
> 
> Here are a few topics I'll suggest please give it a try
> 
> 1. Please go to your start topics and suggest add suggestions on a few topics. based on that suggestion GitHub feed will show you result
> 
> Attaching an image of how it should look. you can easily find the stars repository tab by going to profile, than navigating to stars. ![Screenshot from 2022-06-30 20-01-31](https://user-images.githubusercontent.com/22853459/176704356-9286d862-2e90-499b-9f67-0995f992d312.png)

@surajsahani can u copy paste this quote as a suggested answer, please ?
So I can mark it as a solution. 

It worked after following few topics !

**Reply by surajsahani** [reply]
I'm glad it worked out. @mohaidoss . 
Pasting the same answer below ✌️

**Comment by surajsahani** [comment, marked as answer]
I understand, that there is no new content because of no new activity. it will grow as you increase your activity just exploring more repositories will also work. try that out if it won't work you can create a bug issue on GitHub

Here are a few topics I'll suggest please give it a try

    Please go to your start topics and suggest add suggestions on a few topics. based on that suggestion GitHub feed will show you result

Attaching an image of how it should look. you can easily find the stars repository tab by going to profile, than navigating to stars. 
![Screenshot from 2022-06-30 20-01-31](https://user-images.githubusercontent.com/22853459/177334730-ac167103-75ad-4dc7-adbc-1c8f39fc6280.png)

**Reply by mohaidoss** [reply]
Thanks, very helpful 😃

**Comment by jfullstackdev** [comment]
hello guys, there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)

**Reply by crystalv05264** [reply]
Update All

**Reply by crystalv05264** [reply]
Update Now Please

**Comment by its0x08** [comment]
It still has less information then the "Following" feed.
For example I don't see the events happening towards my repos!
